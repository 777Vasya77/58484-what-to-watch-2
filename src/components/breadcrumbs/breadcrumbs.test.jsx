import React from 'react';
import renderer from 'react-test-renderer';
import Breadcrumbs from '~/components/breadcrumbs/breadcrumbs';

jest.mock(`react-router-dom`, () => ({
  Link: `Link`
}));

describe(`Breadcrumbs component tests`, () => {
  it(`Component render correctly`, () => {
    const tree = renderer
        .create(
            <Breadcrumbs filmId={1} title={`test`}/>
        ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
