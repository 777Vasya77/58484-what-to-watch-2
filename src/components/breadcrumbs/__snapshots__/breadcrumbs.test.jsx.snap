// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumbs component tests Component render correctly 1`] = `
<nav
  className="breadcrumbs"
>
  <ul
    className="breadcrumbs__list"
  >
    <li
      className="breadcrumbs__item"
    >
      <Link
        className="breadcrumbs__link"
        to="/films/1"
      >
        test
      </Link>
    </li>
    <li
      className="breadcrumbs__item"
    >
      <a
        className="breadcrumbs__link"
      >
        Add review
      </a>
    </li>
  </ul>
</nav>
`;
